lib/tests/testscripts/scripts/reporting/submit-data.xml

```diff
@@ -0,0 +1,232 @@
+				<encodeRequestUrl value="false"/>
+				<sourceId value="submit-data-parameters"/>
+			</operation>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the returned HTTP status is 200(OK)."/>
+				<direction value="response"/>
+				<responseCode value="200"/>
+			</assert>
+		</action>
+		<action>
+			<operation>
+				<type>
+					<system value="http://hl7.org/fhir/testscript-operation-codes"/>
+					<code value="evaluate-measure"/>
+				</type>
+				<resource value="Measure"/>
+				<description value="Evaluate a measure."/>
+				<accept value="json"/>
+				<contentType value="json"/>
+				<url value="Measure/${createMeasureId}/$evaluate-measure?patient=${createPatientId}&amp;periodStart=2017&amp;periodEnd=2017"/>
+				<responseId value="measure-calc-report"/>
+			</operation>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the returned HTTP status is 200(OK)."/>
+				<direction value="response"/>
+				<responseCode value="200"/>
+			</assert>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the returned format is JSON. Warning only as the server may not support the return of response content."/>
+				<contentType value="json"/>
+				<warningOnly value="true"/>
+			</assert>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the return type is MeasureReport. Warning only as the server may not support the return of response content."/>
+				<resource value="MeasureReport"/>
+			</assert>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the returned MeasureReport contains the expected initial population."/>
+				<operator value="equals"/>
+				<path value="//fhir:MeasureReport/fhir:group/fhir:population[fhir:code[fhir:coding[fhir:code[@value='initial-population']]]]/fhir:count/@value"/>
+				<value value="1"/>
+			</assert>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the returned MeasureReport contains the expected denominator."/>
+				<operator value="equals"/>
+				<path value="//fhir:MeasureReport/fhir:group/fhir:population[fhir:code[fhir:coding[fhir:code[@value='denominator']]]]/fhir:count/@value"/>
+				<value value="1"/>
+			</assert>
+		</action>
+		<action>
+			<assert>
+				<description value="Confirm that the returned MeasureReport contains the expected numerator."/>
+				<operator value="equals"/>
+				<path value="//fhir:MeasureReport/fhir:group/fhir:population[fhir:code[fhir:coding[fhir:code[@value='numerator']]]]/fhir:count/@value"/>
+				<value value="1"/>
+			</assert>
+		</action>
+	</test>
+
+	<teardown>
+		<action>
+			<operation>
+				<type>
+					<system value="http://hl7.org/fhir/testscript-operation-codes"/>
+					<code value="delete"/>
+				</type>
+				<resource value="Patient"/>
+				<description value="Delete the patient resource on the test server using the id of the Patient."/>
+				<targetId value="test-Patient-294"/>
```

Comment:
The `targetId` does not match the ID of the Patient that's created during `submit-data`. I assume we want these to match up so I'd replace this with the variable `${createPatientId}` that you defined.